src/cmdline.rs

```diff
@@ -438,6 +457,40 @@ pub fn parse_args() -> Result<Config> {
                                 .takes_value(true),
                         ),
                 )
+                .subcommand(
+                    SubCommand::with_name("embed-kargs")
+                        .about("Embed additional kernel args in an ISO image")
```

Comment:
Maybe `Add kernel arguments to the ISO image`?

One concern: we also have the same use case for the ISO of needing to *remove* the default `nosmt` in FCOS?

Reply:
The wording here matches the wording used for the `ignition` subcommand. It's more or less `s/embedded Ignition config/embedded additional kernel args/`. Thought that was nice, but not against rewording entirely if folks prefer!

Re. `nosmt`, yeah right now this is limited to managing *additional* kernel args. I think we could redesign things so that it handles base kargs as well, though it's definitely easier technically (and I think somewhat conceptually too) to operate in an "overlay" pattern, again like the `ignition` command. For example, `appended-kargs remove` will change the file back to a binary match of the original file.

I was looking at the kargs we bake in today, and the two big ones that one might want to revert are the `smt` one and the cgroupsv2 one. Both of those can be done by just overriding with another karg (i.e. `mitigations=off systemd.unified_cgroup_hierarchy=1`).

Reply:
Actually hmm, for the `mitigations=off` test, my test might not be valid because I was testing in a VM with a single core & thread anyway. But I definitely observed `cat /sys/devices/system/cpu/vulnerabilities/*` change output. I'll do another sanity-check that SMT can be re-enabled this way.

Reply:
> I'll do another sanity-check that SMT can be re-enabled this way.

OK, so this test was slightly wrong because the default kernel behaviour is `mitigations=auto`, not `off`. But anyway, I did verify using `qemu-kvm -smp threads=2` and `lscpu` that appending `mitigations=auto` is indeed equivalent to removing the `mitigations=auto,nosmt` flag.